picframe/get_image_meta.py

```diff
@@ -102,7 +103,23 @@ def get_orientation(self):
 
     def get_exif(self, key):
         try:
-            val = self.__get_if_exist(key)
+            iso_keys = ['EXIF ISOSpeedRatings', 'EXIF PhotographicSensitivity', 'EXIF ISO'] # ISO prior 2.2, ISOSpeedRatings 2.2, PhotographicSensitivity 2.3
+            if key in iso_keys:
+                for iso in iso_keys:
+                    val = self.__get_if_exist(iso)
+                    if val:
+                        break
+            else:
+                val = self.__get_if_exist(key)
+
+            if val is None:

```

Comment:
This looks like it should work to cascade between the two groups, so maybe this is all we need for now. It might be nice (for future) to have something like `get_exif_fuzzy` (or similar) that used a dictionary keyed by a generic tag name with the value being a list of possible group/tag names to search for a match. So, something like this, for example:

`{'make': ['Image Make', 'EXIF Make'], 'model': ['Image Model', 'EXIF Model]}`